AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/TaskController.cs

```diff
@@ -92,6 +92,7 @@ public TaskController(IMediator mediator, ITaskEditViewModelValidator taskEditVi
                     RedirectToAction(nameof(Details), "Task", new { id = taskId });
             }
 
+            viewModel.CancelUrl = Url.Action(new UrlActionContext { Action = nameof(Details), Controller = "Task", Values = new { eventId = viewModel.EventId, id = viewModel.Id, area = "Admin" } });

```

Comment:
I spent a little time looking at this @mjmilan and this line is where an exception is thrown during the three failing tests I mentioned earlier.

Specifically when `Url.Action.(..)` is called an `Object reference not set to instance` exception occurs. @stevejgordon have you seen this before?